confidant/routes/services.py

```diff
@@ -285,12 +285,19 @@ def map_service_credentials(id):
         except keymanager.ServiceCreateGrantError:
             msg = 'Failed to add grants for {0}.'.format(id)
             logging.error(msg)
+    credentials = credentialmanager.get_credentials(data.get('credentials'))
```

Comment:
this probably won't happen but does any service have >100 credentials?  The docs say that BatchGet returns at most 100 items

Reply:
True. It's unlikely any have >100, but we should probably do a check to see if there's unprocessed records, and if so, fetch those as well. Not sure if pynamo handles that for you or not. In either case, it's out of scope for this PR, since it's a general problem in the software.

Reply:
Docs say that pynamo handles batch limits invisibly: https://pynamodb.readthedocs.io/en/latest/batch.html

Reply:
I saw that doc but the code doesn't seem like it uses UnprocessedKeys https://github.com/pynamodb/PynamoDB/blob/master/pynamodb/connection/base.py#L1112.

If we don't have any service having >100 keys and the chances of one having >100 in the future is small , it might be worth just adding a stat and alarming on the stat.  You can implement pagination when needed.

Reply:
Hm. That's good to know. I'm not going to work that into this PR, but will address this in a followup, since we need to handle this case everywhere, not just the script. I'll probably add a wrapper function for batch gets that handles this, that we call for for all batch gets.

Reply:
It does; it's in the model class: https://github.com/pynamodb/PynamoDB/blob/master/pynamodb/models.py#L321-L332